server/src/main/java/com/google/coffeehouse/servlets/GetProfileServlet.java

```diff
@@ -65,44 +89,52 @@ public GetProfileServlet(StorageHandlerApi storageHandler) {
   public GetProfileServlet() {
     super();
     this.storageHandler = new StorageHandlerApi();
+    this.verifier = new GoogleIdTokenVerifier.Builder(transport, jsonFactory).build();
   }
   
   /** 
    * Returns a {@link Person} object in JSON format from information in the database.
-   * @param request the GET request that must have a valid JSON representation of the userId to be
-   *     passed in order to fetch a person from ID in the database.  If the required "userId"
-   *     field doesn't exist, the response object will send a "400 Bad Request error". If the JSON
+   * @param request the GET request that must have a valid JSON representation of the
+   *     {@code "idToken"} of the user to be passed in order to fetch a person from ID in the
+   *     database.  If the required "userId" field doesn't exist, the response object will
```

Comment:
Nit: This needs to be updated because there is no "userId" field, only the "idToken" field from which we can generate the userId.